pants.toml

```diff
@@ -0,0 +1,52 @@
+[GLOBAL]
+pants_version = "2.7.1"
+backend_packages = [
+    "pants.backend.shell",
+    "pants.backend.shell.lint.shellcheck",
+    "pants.backend.shell.lint.shfmt"
```

Comment:
I noticed that the Dockerfile here has a hadolint ignore comment. If that's the case it may make sense to enable the docker and hadolint backends for pants. Not necessary for this PR though, it can be done in a follow-up PR

Reply:
I'll look into this as a follow-up PR.